README.md

```diff
@@ -261,6 +263,7 @@ export default combine(
   jsonc(),
   yaml(),
   toml(),
+  xml(),
```

Comment:
This is not true 😅

Reply:
Damn, you're right :D it used to be true but then I changed it. Good catch.